src/chocolatey/infrastructure.app/services/ChocolateyPackageInformationService.cs

```diff
@@ -266,7 +266,7 @@ private static string GetStorePath(IFileSystem fileSystem, string id, NuGetVersi
                 return preferredStorePath;
             }
 
-            var pkgStorePath = fileSystem.CombinePaths(ApplicationParameters.ChocolateyPackageInfoStoreLocation, "{0}.{1}".FormatWith(id, version.ToStringSafe()));
+            var pkgStorePath = fileSystem.CombinePaths(ApplicationParameters.ChocolateyPackageInfoStoreLocation, "{0}.{1}".FormatWith(id, version.ToNormalizedStringChecked()));

```

Comment:
This one should not be `ToNormalizedStringChecked`. it uses `ToStringSafe` on purpose to catch legacy installations of a package.

Reply:
Can we add a comment in this area, so that this information isn't lost?